mdata/cache/ccache_metric.go

```diff
@@ -202,13 +212,13 @@ func (mc *CCacheMetric) searchBackward(from, until uint32, keys []uint32, res *C
 	}
 
 	for ; ts != 0; ts = mc.chunks[ts].Prev {
+		if ts < from {
+			break
+		}
+
 		log.Debug("CCacheMetric searchBackward: backward search adds chunk ts %d to end", ts)
 		res.End = append(res.End, mc.chunks[ts].Itgen)
 		res.Until = ts
-
-		if ts <= from {
-			break
-		}
```

Comment:
Why this change? is it related to setting prev on cache add? if yes, how so? if not, then what is this? is this a bugfix? what's the bug?